Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-updateSearchIndex.txt

MongoDB\\Collection::updateSearchIndex()
========================================

.. meta::

>    :description: Update an Atlas Search index for a collection using `MongoDB\Collection::updateSearchIndex()` in PHP, requiring an Atlas cluster tier of at least M10.

Reviewer: ```suggestion
   :description: Update an Atlas Search index for a collection with the MongoDB PHP Library, requiring an Atlas cluster tier of at least M10.
```
